crown-court-contribution/src/test/java/uk/gov/justice/laa/crime/contribution/service/MaatCourtDataServiceTest.java

```diff
@@ -133,5 +134,22 @@ void givenAInvalidParameter_whenGetRepOrderCCOutcomeByRepIdIsInvoked_thenReturnE
         ).isInstanceOf(APIClientException.class);
     }
 
+    @Test
+    void givenAValidRepId_whenGetContributionsSummaryIsInvoked_thenContributionsSummariesAreReturned() {
+        maatCourtDataService.getContributionsSummary(TEST_REP_ID, LAA_TRANSACTION_ID);
+
+        verify(maatCourtDataClient).get(eq(new ParameterizedTypeReference<List<ContributionsSummaryDTO>>() {}),
+                anyString(), anyMap(), anyInt());
+    }
+
+    @Test
+    void givenInvalidResponseFromMaatApi_whenGetContributionsSummaryIsInvoked_thenExceptionIsRaised() {
+        when(maatCourtDataClient.get(any(), anyString(), anyMap(), anyInt())).thenThrow(new APIClientException());
+
+        assertThatThrownBy(() -> maatCourtDataService.getContributionsSummary(
+                TEST_REP_ID, LAA_TRANSACTION_ID)
+        ).isInstanceOf(APIClientException.class);
+    }
```

Comment:
I have to say, I'm not sure this test or the one on line 129-135 are adding anything. The exception handling is all external to the class under test.

Reply:
Fair enough that makes sense, I've removed both exception handling tests.